runtime/src/main/java/org/corfudb/protocols/wireprotocol/SegmentSizeRequest.java

```diff
@@ -0,0 +1,27 @@
+package org.corfudb.protocols.wireprotocol;
+import io.netty.buffer.ByteBuf;
+import lombok.Getter;
+
+public class SegmentSizeRequest implements ICorfuPayload<SegmentSizeRequest> {
+    @Getter
+    long startAddress;
+    @Getter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4685528)